### PR TITLE
CreateML fixes

### PR DESCRIPTION
**Pull request by saitejamalyala**
- write verified field to the json file while saving the annotation in createMl format
- open annotation in json format

### CODE REVIEW

File: labelImg.py [hunk 1]
```diff
@@ -1196,10 +1197,15 @@ def show_bounding_box_from_annotation_file(self, file_path):
         else:
             xml_path = os.path.splitext(file_path)[0] + XML_EXT
             txt_path = os.path.splitext(file_path)[0] + TXT_EXT
+            json_path = os.path.splitext(file_path)[0] + JSON_EXT
+
             if os.path.isfile(xml_path):
                 self.load_pascal_xml_by_filename(xml_path)
             elif os.path.isfile(txt_path):
                 self.load_yolo_txt_by_filename(txt_path)
+            elif os.path.isfile(txt_path):
```
Comment (tzutalin) [inline labelImg.py:1206]:
```
    elif os.path.isfile(txt_path):
                self.load_yolo_txt_by_filename(txt_path)
    elif os.path.isfile(txt_path):
                self.load_create_ml_json_by_filename(json_path, file_path)
```
The second `elif` will never happen

File: labelImg.py [hunk 2]
```diff
@@ -1300,10 +1306,13 @@ def change_save_dir_dialog(self, _value=False):
         if dir_path is not None and len(dir_path) > 1:
             self.default_save_dir = dir_path
 
+        self.show_bounding_box_from_annotation_file(self.file_path)
```
Comment (tzutalin) [inline labelImg.py:1309]: why need this?

File: libs/create_ml_io.py [hunk 3]
```diff
@@ -32,6 +32,7 @@ def write(self):
 
         output_image_dict = {
             "image": self.filename,
+            "verified": True if self.verified else False,
```
Comment (tzutalin) [inline libs/create_ml_io.py:35]: Looks verbose.

`  "verified": True if self.verified else False,`

can be simplified to `"verified":  self.verified`?

File: libs/create_ml_io.py [hunk 4]
```diff
@@ -108,7 +109,8 @@ def parse_json(self):
             input_data = file.read()
 
         output_dict = json.loads(input_data)
-        self.verified = True
+        
+        self.verified = output_dict[0].get("verified", False)
```
Comment (tzutalin) [inline libs/create_ml_io.py:113]: `output_dict[0]`? why accessing 0 index from a dict?

File: labelImg.py (hunk 1 above)
Comment (saitejamalyala) [reply labelImg.py]: fixed this, Last elif should be 
` elif os.path.isfile(json_path):
                self.load_create_ml_json_by_filename(json_path, file_path)`

File: labelImg.py [hunk 5]
```diff
@@ -90,7 +90,7 @@ def __init__(self, default_filename=None, default_prefdef_class_file=None, defau
 
         # Save as Pascal voc xml
         self.default_save_dir = default_save_dir
-        self.label_file_format = settings.get(SETTING_LABEL_FILE_FORMAT, LabelFileFormat.PASCAL_VOC)
+        self.label_file_format = settings.get(SETTING_LABEL_FILE_FORMAT, LabelFileFormat.CREATE_ML)
```
[the remark this answers is not in the document]
Comment (saitejamalyala) [reply labelImg.py:93]: Yes, For sure

File: labelImg.py (hunk 2 above)
Comment (saitejamalyala) [reply labelImg.py:1309]: if there are existing annotations in the new directory, it shows the bbox annotations for the current file ?

File: libs/create_ml_io.py (hunk 3 above)
Comment (saitejamalyala) [reply libs/create_ml_io.py:35]: Ofc, fixed now

File: libs/create_ml_io.py (hunk 4 above)
Comment (tzutalin) [inline libs/create_ml_io.py:113]: One more question. What is output_dict? is it always expected to have a value in the list? should it have a empty check before assigning to self.verified like

` output_dict[0].get("verified", False) if len(output_dict) else False`

**Review by tzutalin — COMMENTED** [review]
One more comment about output_dict

**Pull request merged**